src/main/java/com/alkemy/ong/web/controller/SlidesController.java

```diff
@@ -0,0 +1,77 @@
+package com.alkemy.ong.web.controller;
+
+import com.alkemy.ong.domain.slides.model.Slides;
+import com.alkemy.ong.domain.slides.model.response.SlidesResponse;
+import com.alkemy.ong.domain.slides.services.SlidesService;
+import lombok.*;
+import org.springframework.beans.BeanUtils;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/slides")
+public class SlidesController {
+    private final SlidesService slidesService;
+
+    public SlidesController(SlidesService slidesService) {
+        this.slidesService = slidesService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<SlidesResponse>> getSlides(){
```

Comment:
en lugar de slidesResponse prefiero q retornes la lista de slide dtos. 
Si hacemos eso, te evitas el ultimo bloque:
for(SlidesDto slidesDto : slides){
             SlidesResponse slidesModel = new SlidesResponse();
             BeanUtils.copyProperties(slidesDto, slidesModel);
             returnValue.add(slidesModel);
         }

Reply:
Igualemente tendría que hacerlos pasar por un método (llámese `toResponse`) que solo me devuelva lo que pide el ticket, verdad? Es decir, que devuelva la imagen y el orden nada más.

Reply:
ah cierto q el ticket no pide todo. Déjalo asi entonces!